### ISSUE TITLE
Show rich text?

### ISSUE THREAD

**Issue by MindstoneMedia** [overtrue/laravel-versionable#63]
Is it possible to show rich text content without the html tags? I have to compare a content field from a rich text editor.

**Comment by mansoorkhan96** [comment]
@MindstoneMedia Is this for the diff feature?

**Comment by MindstoneMedia** [comment]
Yes, for the two fields where the user can compare the versions > see screenshot.
<img width="820" alt="Bildschirmfoto 2024-04-24 um 17 58 52" src="https://github.com/overtrue/laravel-versionable/assets/74834981/634b5264-544b-4d10-ab41-d005712cb44b">

**Comment by mansoorkhan96** [comment]
Let me try to PR it.

Oh just realised you are using my filament plugin. nice.

**Comment by mansoorkhan96** [comment]
This feature is now available in the latest release and also has been added to Filament Versionable.
